server/services/premisesService.ts

```diff
@@ -14,9 +14,9 @@ import { mapApiOccupancyToUiOccupancy } from '../utils/premisesUtils'
 export default class PremisesService {
   constructor(private readonly premisesClientFactory: RestClientBuilder<PremisesClient>) {}
 
-  async getAll(token: string): Promise<Array<ApprovedPremisesSummary>> {
+  async getAll(token: string, probationRegionId = ''): Promise<Array<ApprovedPremisesSummary>> {
     const premisesClient = this.premisesClientFactory(token)
-    const premises = await premisesClient.all()
+    const premises = await premisesClient.all(probationRegionId)
```

Comment:
We don't seem to use this anywhere?

Reply:
yeah my bad, I'll use the API side filtering

Reply:
I've deleted the Cypress test for the filtering (that I had previously added) but will write an E2E for it